notebooks/Notebook3.md

```diff
@@ -2,33 +2,66 @@
+Currently, I have it as a list with each element representing a transition and each transition containing a list the elements and effects to apply.
+I have all the transitions simplified into three types now: adding/removing classes, adding/removing transformations (translations, rotations, scaling, stretching), and changing element styles
+
+With both parsers working, I was able to replicate my presentation from last week into a [copy](<https://hmc-cs111-spring2023.github.io/project-code-occamkg/work%20files/testPres0/output>) produced by my program from the code.
+I also added a new [test presentation](<https://hmc-cs111-spring2023.github.io/project-code-occamkg/work%20files/testPres1/presentation>)!
+Some things I wanted to show with this are translating and scaling objects (images in this case), adding and removing classes (the blur added/removed from the text), and full size background images (the puppy!).
+The source input for this presentation can be found [here](<https://github.com/hmc-cs111-spring2023/project-code-occamkg/tree/master/work%20files/testPres1>) with the output in the `presentation` folder.
+I'm pretty satisfied with the transition syntax, but the element syntax is a bit bulky, so I may look into simplifying it further.
+I also added the slide number with some navigation controls, so that will be on every presentation.
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+Currently, whenever I want to render a change, I first have to save my file and then run the command `python path/to/program/run.py`.
+VS Code already has a very convenient feature (Open with Live Server) that rerenders HTML pages whenever you save anything in the workspace,
+but since my code doesn't output the HTML when it's saved, I can't take advantage of this.
+Ideally, what I would like is for the user to be able to choose/make a folder for their presentation and then have the code generate blank `.yaml`, `.css`, and `.frame` files.
+Then, whenever any files in this folder are saved, it would output the resulting web files.
+I'm fine with the user needing to install additional software (such as Python), but I would like this to function on its own once those are installed.
+
+Additionally, I'm always looking for more features to add.
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+- I'm always open to more feature suggestions since the idea of this DSL is to allow more flexibility than what is in most presentation softwares, so probably a lot of things I haven't even considered.
+
+- If there are any suggestions for how I could make the element code ([example](<https://github.com/hmc-cs111-spring2023/project-code-occamkg/blob/master/work%20files/testPres1/elements.yaml>)) more readable, I would love to hear it.
+
```

Comment:
Perhaps implicit, default values? For example, if the `class` for `main-title` is almost always going to be `title`, your implementation could provide a default value for `class`, so that the user doesn't need to write it.